### PR TITLE
Improve casting local files/subtitles and improve mobile remote view

### PR DESCRIPTION
**Pull request by mheine**
- The main changes I've done are in rcast.py, where there is now more thorough logging and configuration.
 - The configuration file has a few new options, most notably "width" and "height", which can (optionally) be used to set a certain resolution for the omxplayer.
 - The remote view has been updated with separated buttons and a clear-option, which drastically improves the mobile experience.
- The readme has been slightly restructed and has received a few grammatical fixes.

Feel free to point out anything which you would like to see done differently, or if there are any other questions.

### CODE REVIEW

**Comment by vincelwt** [comment]
Hey! First, thanks a lot :) A little fresh air for RespberryCast :P

I added a comments on a few points

**Comment by vincelwt** [comment]
Hey! First, thanks a lot :) A little fresh air for RespberryCast :P

I added a comments on a few points

**Comment by mheine** [comment]
I can't seem to find any comments, am I missing something? 😅

**Comment by vincelwt** [comment]
Oops, forgot to explicitly start the review.

File: rcast.py [hunk 1]
```diff
@@ -1,32 +1,97 @@
...
+if(len(sys.argv) > 3 or len(sys.argv) < 2):
+	log("Incorrect number of arguments given. Program expects 1-2 arguments.\nA playable video file, and an optional subtitle file.")
+	sys.exit(0)
+
+# Attempt to load and configuration file
+try:
+
+	print(sys.path[0])
+
+	with open(sys.path[0] + "/raspberrycast.conf") as f:
```
[the remark this answers is not in the document]
Comment (mheine) [reply rcast.py]: Aye, raspberry.local:2020 works, assuming the user hasn't changed the hostname of their pi (which I had - I wasn't quite sure how the .local-addresses worked and this seemed easier to me). Perhaps some sort of connection check? Defaulting to raspberrrypi.local, unless there is another IP set in the config?

The whole config file isn't needed, but how about supplying changing the config-file to include pi_ip, height and width, but not give them a value? This means that the user can, if necessary change the values to what they desire, and the code can check if the value is not ""?

[the remark this answers is not in the document]
Comment (mheine) [reply rcast.py]: If you accept the changes proposed above, do you want them as an addition to this PR, or a separate one? :)

[the remark this answers is not in the document]
Comment (vincelwt) [reply rcast.py]: Yep I agree with that :) Config shouldn't be mandatory (and default to what the original settings were) but can be useful for advanced users with different ips / who want to change the resolution. 
You can add the changes as part of this PR ;)

**Comment by mheine** [comment]
Updated files to better handle default values when no configuration file is given. Should also work if the user (for some reason) has an older version of the config file.

**Comment by vincelwt** [comment]
Thanks a bunch :)

**Pull request merged**